cypress.config.js

```diff
@@ -0,0 +1,12 @@
+const { defineConfig } = require("cypress");
+module.exports = defineConfig({
+  "env": {
+    baseUrl : 'https://dev.urbuddi.com/login'
+  },
+
+  e2e: {
```

Comment:
Hi @karthikoptim 
can you remove this **e2e** block?

Reply:
With out e2e block in configuration, we con not exicute script